test/passing/break_struct.ml.ref

```diff
@@ -1,3 +1,35 @@
 [@@@ocamlformat "break-struct=natural"]
 
 module M = X (Y) (struct let x = k end)
+
+module Hash = struct include Hash module type S = S.HASH end
+
+module Hash = struct
+  include Hash include Hash module type S = S.HASH module type S = S.HASH
+end
+
+module Hash = struct
+  include Hash
+  include Hash
+  include Hash
+
+  module type S = S.HASH
+
```

Comment:
why does it adds a newline between the `module type S` sentences?

Reply:
This is due to the grouping (similar simple items are grouped together) that is done for structure items. Otherwise, everything else seems to be formatted alright